src/System.Memory/tests/Span/CopyTo.cs

```diff
@@ -204,6 +204,12 @@ public static void CopyToCovariantArray()
         [InlineData((4L * 1024L * 1024L * 1024L) + 256)]
         public static void CopyToLargeSizeTest(long bufferSize)
         {
+            // If this test is run in a 32-bit process, the large allocation will fail.
+            if (Unsafe.SizeOf<IntPtr>() != sizeof(long))
```

Comment:
It's just more convoluted, and involves unnecessary usage of `Unsafe`. It also relies on an unclear relationship between the size of `IntPtr` and the size of `Int64`.

I think we use `IntPtr.Size == 8` in other parts of corefx, but if this pattern is more consistent with the Span tests, then it's okay.